public/js/app.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+const mylist = new ShoppingList();
+const shoppingListButton = document.getElementById("addShoppingListItemButton");
+shoppingListButton.addEventListener("click", addToShoppingList);
+const content = document.getElementById("content");
+const itemName = document.getElementById("name");
+const itemDescription = document.getElementById("description");
+
+function addToShoppingList(event) {
+  var newShoppingListItem = new ShoppingListItem(
+    itemName.value,
+    itemDescription.value
+  );
+  mylist.addItem(newShoppingListItem);
+  content.innerHTML = mylist.render(newShoppingListItem);
+}
+
+function changeCheckedStatus(i, checkbox) {
```

Comment:
Instead of having this function that goes and checks or unchecks each box. You could update the ShoppingListItem's render function to render a input that's already checked.